### ISSUE TITLE
Crash on attempting to skip championship finish animation

### ISSUE THREAD

**Issue by Eiim** [agateau/pixelwheels#118]
On Windows, if I attempt to skip past the championship ending animation too early, the game crashes with this exception:

```
Exception in thread "main" java.lang.IllegalArgumentException: The same task may not be scheduled twice.
        at com.badlogic.gdx.utils.Timer.scheduleTask(Timer.java:80)
        at com.badlogic.gdx.utils.Timer.postTask(Timer.java:62)
        at com.badlogic.gdx.utils.Timer.post(Timer.java:167)
        at com.agateau.pixelwheels.sound.MusicFader.fadeOut(MusicFader.java:53)
        at com.agateau.pixelwheels.sound.DefaultAudioManager.fadeOutMusic(DefaultAudioManager.java:121)
        at com.agateau.pixelwheels.sound.DefaultAudioManager.playMusic(DefaultAudioManager.java:102)
        at com.agateau.pixelwheels.PwGame.showMainMenu(PwGame.java:146)
        at com.agateau.pixelwheels.gamesetup.ChampionshipMaestro$5.lambda$onNextPressed$0(ChampionshipMaestro.java:182)
        at com.agateau.pixelwheels.gamesetup.Maestro.showUnlockedRewardScreen(Maestro.java:96)
        at com.agateau.pixelwheels.gamesetup.Maestro.showUnlockedRewardScreen(Maestro.java:90)
        at com.agateau.pixelwheels.gamesetup.ChampionshipMaestro$5.onNextPressed(ChampionshipMaestro.java:182)
        at com.agateau.pixelwheels.screens.NavStageScreen.onNextPressed(NavStageScreen.java:86)
        at com.agateau.pixelwheels.screens.NavStageScreen$2.clicked(NavStageScreen.java:73)
        at com.badlogic.gdx.scenes.scene2d.utils.ClickListener.touchUp(ClickListener.java:88)
        at com.badlogic.gdx.scenes.scene2d.InputListener.handle(InputListener.java:59)
        at com.badlogic.gdx.scenes.scene2d.Stage.touchUp(Stage.java:351)
        at com.badlogic.gdx.InputEventQueue.drain(InputEventQueue.java:86)
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Input.update(Lwjgl3Input.java:205)
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Window.update(Lwjgl3Window.java:390)
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.loop(Lwjgl3Application.java:137)
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:111)
        at com.agateau.pixelwheels.desktop.DesktopLauncher.main(DesktopLauncher.java:37)
```

**Comment by agateau** [comment]
Thanks for the report (and the backtrace! 👍🏻), going to look into it.

**Comment by agateau** [comment]
Can you try this build: https://builds.agateau.com/pixelwheels/pixelwheels-0.19.1+20213105T172442-cf8fb33-windows64.zip ? It should fix this crash and your other crash (#120)

**Comment by Eiim** [comment]
Can confirm both crashes are fixed!
